checklist.md

```diff
@@ -0,0 +1,103 @@
+# Сheck Your Code Against the Following Points
+
+## Code Efficiency
+
+1. Prevent creating a redundant list via passing comprehension without square brackets when needed.
+
+Good example:
+
+```python
+def get_sum(numbers: list) -> int | float:
+    return sum(number for number in numbers)
+```
+
+Bad example:
+
+```python
+def get_sum(numbers: list) -> int | float:
+    return sum([number for number in numbers])
+```
+
+2. Do not repeat abstract methods from parent if you have an abstract class that inherits another abstract class.
```

Comment:
What sense? I don't understand this option

Reply:
https://github.com/mate-academy/py-elves-and-dwarves/pull/8
https://github.com/mate-academy/py-elves-and-dwarves/pull/9
https://github.com/mate-academy/py-elves-and-dwarves/pull/12
there are a lot of mistakes like these

Reply:
what about [this](https://github.com/mate-academy/py-elves-and-dwarves/pull/4/files)?